src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -89,11 +93,25 @@ public boolean getGyroReset() {
 	
 	public double getArcadeSpeed() {
 		return getDriveJoystick().turboSpeedAdjust();
-		
 	}
 	
 	public double getArcadeTurn() {
 		return getDriveJoystick().getRightStickX();
 	}
-
+	
+	public void setLeftRumble(double value) {
+		getDriveJoystick().setRumble(RumbleType.kLeftRumble, value);
+	}
+	
+	public void setRightRumble(double value) {
+		getDriveJoystick().setRumble(RumbleType.kRightRumble, value);
+	}
+	
+	public boolean leftRumbleButtonDown() {
+		return getDriveJoystick().buttonDown(Button.x);
+	}
+	
+	public boolean rightRumbleButtonDown() {
+		return getDriveJoystick().buttonDown(Button.b);
+	}
```

Comment:
These "rumble buttons" were just for test, right? We shouldn't keep that on master, should we?